src/scripts/main.js

```diff
@@ -1,3 +1,42 @@
 'use strict';
 
-// write code here
+const employeeList = document.querySelector('ul');
+
+const parseSalary = (salaryStr) => {
+  let cleanStr = salaryStr.split('$')[1];
```

Comment:
Ensure that the salary string always contains a '$' symbol. If there's a possibility of missing or malformed salary strings, consider adding error handling to manage such cases.